dsgrid/dataset/dataset_schema_handler_one_table.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, load_data_df, *args, **kwargs):
 
     @classmethod
     def load(cls, config: DatasetConfig, *args, **kwargs):
-        path = Path(config.model.path)
+        path = Path(config.dataset_path)
         load_data_df = read_dataframe(check_load_data_filename(path))
```

Comment:
Can we just use `load_data_df = read_dataframe(config.load_data_path)` here to be consistent with standard schema_handler?